tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/model/Type.java

```diff
@@ -22,7 +22,15 @@
  */
 public enum  Type {
     /**
-     * Hawkular APM environment (requires instrumentation of the application).
+     * Hawkular APM Agent environment (requires instrumentation of the application using java agent).
+     */
+    APMAGENT,
+    /**
+     * Hawkular APM OpenTracing Agent environment (requires instrumentation of the application using java agent).
```

Comment:
Change this to something like `Will instrument application with Hawkular opentracing agent.`